Fitness App/Models/MapPin.m

```diff
@@ -19,18 +19,18 @@ @implementation MapPin
 @synthesize subtitle;
 @synthesize coordinate;
 
--(MKAnnotationView *) annotationView{
-    MKAnnotationView *annotationView = [[MKAnnotationView alloc] initWithAnnotation:self reuseIdentifier:@"MyCustomAnnotation"];
-    annotationView.enabled = YES;
-    annotationView.canShowCallout = YES;
-    annotationView.image = self.image;
-    UIButton *infoButton = [UIButton buttonWithType:UIButtonTypeDetailDisclosure];
-    annotationView.rightCalloutAccessoryView = infoButton;
-
-    [infoButton addTarget:self action:@selector(buttonPressed:) forControlEvents:UIControlEventTouchUpInside];
-    
-    return annotationView;
-}
+//-(MKAnnotationView *) annotationView{
+////    MKAnnotationView *annotationView = [[MKAnnotationView alloc] initWithAnnotation:self reuseIdentifier:@"MyCustomAnnotation"];
+////    annotationView.enabled = YES;
+////    annotationView.canShowCallout = YES;
+////    annotationView.image = self.image;
+////    UIButton *infoButton = [UIButton buttonWithType:UIButtonTypeDetailDisclosure];
+////    annotationView.rightCalloutAccessoryView = infoButton;
+////
+////    [infoButton addTarget:self action:@selector(buttonPressed:) forControlEvents:UIControlEventTouchUpInside];
+////
+////    return annotationView;
+//}
```

Comment:
Maybe delete these commented-out lines if you don't need them anymore?

Reply:
nice point, actually I don't use anything in that file any more so I will delete the entire thing!